Assets/Scripts/Spawning/Spawner.cs

```diff
@@ -60,6 +84,26 @@ public HashSet<GameObject> SpawnEnemies()
             }
         }
 
-        return spawned;
+        spawningComplete?.Invoke(spawned);
+        HideSpawnPoints();
+    }
+
+
+    /* Disables each spawn point so it no longer appears in game.
+     */ 
+    private void HideSpawnPoints()
+    {
+        foreach(SpawnPoint spawnPoint in spawnPoints)
+        {
+            spawnPoint.gameObject.SetActive(false);
```

Comment:
Why not just destroy it?

Reply:
In case we need it later for run-time spawning, idk :D